kolibri_explore_plugin/assets/src/customApps.js

```diff
@@ -281,6 +282,7 @@ export const ChannelCardIcons = {
   '057f871caa405ec29d62ba0523c193d7': 'music.png',
   '914fee213ee146de869016c287116b23': 'novels.png',
   '59bb2e5a3d2e5e3b85b87d9ab4daa2f3': 'reading.png',
+  a8e6591f1afa426d859318a0a29d1237: 'samhsa.png',
```

Comment:
I think the hash needs to be quoted, `'a8e6591f1afa426d859318a0a29d1237'`, here too